cypress/e2e/modals.cy.ts

```diff
@@ -83,6 +83,21 @@ describe("modals", () => {
       });
     });
 
+    it("add new pmtiles source", () => {
+      let sourceId = "pmtilestest";
+      when.setValue("modal:sources.add.source_id", sourceId);
+      when.select("modal:sources.add.source_type", "pmtiles_vector");
+      when.setValue("modal:sources.add.source_url", "https://data.source.coop/protomaps/openstreetmap/v4.pmtiles");
+      when.click("modal:sources.add.add_source");
+      when.wait(200);
+      then(
+        get.styleFromLocalStorage().then((style) => style.sources[sourceId])
+      ).shouldInclude({
+        type: "vector",
+        // url: "pmtiles://https://data.source.coop/protomaps/openstreetmap/v4.pmtiles",
```

Comment:
Looks like some assertion is missing here to properly test this...

Reply:
Apparently the `onChange` trigger code is not executed in Cypress, so `url` can't be tested for a new value. Someone more knowledgeable about Cypress maybe knows a workaround?

Reply:
CC: @ShellyDCMS, any chance you can take a look at this?

Reply:
@HarelM , checking